fuel-core-interfaces/src/signer.rs

```diff
@@ -3,7 +3,7 @@ use async_trait::async_trait;
 use thiserror::Error;
 
 /// Dummy signer that will be removed in next pull request.
-/// TODO do not use.
+/// TODO: Do not use.
```

Comment:
Unrelated to this PR as well, but isn't a bit weird that we have a public trait that should not be used? Shouldn't we make it private or something?